pkg/pillar/containerd/containerd.go

```diff
@@ -493,8 +493,6 @@ func (client *Client) CtrGetContainerMetrics(ctx context.Context, containerID st
 // For tasks that are in the running, pausing or paused state the PID is also provided
 // and the exit code is set to 0. For tasks in the stopped state, exit code is provided
 // and the PID is set to 0.
-
-// cperakis sos
```

Comment:
It would be nice to amend the change to the original commit, avoiding a list of hot-fix commits.